libs/tof-lib/src/lib/constraint-manager.ts

```diff
@@ -328,35 +328,46 @@ export class ConstraintManager {
    * @param elementTreeModels The tree models to associate to constraints
    */
   private associate(elementTreeModels: ElementTreeModel[]) {
-    for (const elementTreeModel of elementTreeModels) {
-      if (elementTreeModel.constrainedElement) continue;
-
-      const base = elementTreeModel.baseElement;
-      const sliceCounts: { [ path: string ]: number } = {};
-
-      for (const diff of this.structureDefinition.differential.element) {
-        if (!base.path || !base.id || !diff.path || !diff.id) continue;
-
-        const baseId = ConstraintManager.normalizePath(base.id);
-        const diffId = ConstraintManager.normalizePath(diff.id);
-        const idMatch = baseId === diffId;
-        const baseIdDepth = baseId.split(/[\.\/]/g).length;
-        const diffIdDepth = diffId.split(/[\.\/]/g).length;
-        const isNewSlice = diffId.startsWith(baseId + ':') && diffIdDepth === baseIdDepth;
-        const isReSlice = diffId.startsWith(baseId + '/') && diffIdDepth-1 === baseIdDepth;
-
-        // TODO: support multiple re-slices later, after we have conferred with FHIR WG@HL7
-        if ((idMatch || isReSlice) && !elementTreeModel.constrainedElement) {
-          // This is a constraint on something in the base
-          elementTreeModel.constrainedElement = diff;
-        } else if (isNewSlice) {
+    //const baseInfo = this.base.snapshot.element.map(e => e.id + ' - ' + e.path);
+    //console.log(JSON.stringify(baseInfo, null, '\t'));
```

Comment:
we can probably removed these commented out lines if not needed

Reply:
Agreed